lib/manageiq/appliance_console/external_auth_options.rb

```diff
@@ -152,17 +151,8 @@ def self.configured?
 
     def load_current
       say("\nFetching external authentication options from appliance ...")
-      result = ManageIQ::ApplianceConsole::Utilities.rake_run("evm:settings:get", EXT_AUTH_OPTIONS.keys)
-
-      if result.success?
-        return parse_response(result)
-      else
-        raise parse_errors(result).join(', ')
-      end
-    end
-
-    def parse_errors(result)
-      result.error.split("\n").collect { |line| line if line =~ /^error: /i }.compact
```

Comment:
Not sure why they were doing `collect` with an `if` and a `.compact` rather than a `select` :shrug: